app/views/users/registrations/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="text-center mt-4">
```

Comment:
please add this before this line of code:
`<%= provide(:title, "Edit Profile") %>`